src/main/scala/com/cognite/spark/datasource/CdpConnector.scala

```diff
@@ -96,6 +114,13 @@ trait CdpConnector {
     getWithCursor(config, url, initialCursor)
       .flatMap(_.chunk)
 
+  def postWithBody[A: Decoder](config: RelationConfig, url: Uri, body: String): Iterator[A] =
```

Comment:
Should body be a string or should it be a generic type with an encoder?

Reply:
Isn't this `post` except with `DataItems[A]` instead of `Items[A]`? Which has `[A: Encoder]`.

Reply:
Do we actually need and use that return value, though?

Reply:
How would we otherwise return the data that we receive from the request? The return value is used similarly to the way it is used by `get`.

Reply:
Oh, right. Of course.